package.json

```
@@ -5,8 +5,9 @@
   "main": "js/src/index.js",
   "module": "js/src/index.js",
   "scripts": {
-    "test": "NODE_ENV=test ./node_modules/mocha/bin/mocha --require babel-core/register --slow 100 --recursive ./js/**/*.test.js",
-    "prepush": "npm tes"
+    "build": "webpack --color --config webpack.config.js",
+    "test": "NODE_ENV=test ./node_modules/mocha/bin/mocha --require ./js/src/testInit.js --recursive ./js/**/*.test.js",
```

Pas besoin de mettre la path vers le binary. Yarn modifie le path correctement avant de rouler les scripts.

```suggestion
    "test": "NODE_ENV=test mocha --require ./js/src/testInit.js --recursive ./js/**/*.test.js",
```